### DISCUSSION TITLE
Protect wider content

### DISCUSSION THREAD

**Discussion by alpha-pix** [snicco/fortress#3] [Feature Requests]
Key areas to protect content within WP would be form submissions, such as Gravity Forms data from certain roles being able to view it.
And it's a little far fetched, but protecting customer data, even WooCommerce bulk data while cloning or pushing to staging would make it possible to be worked on by people without a GDPR NDA.

**Comment by calvinalkan** [comment]
We might expand V&P to other "data types" where possible. 

But we won't do permission management for third party plugins. That's out of scope.
